redditdownload/gfycat.py

```diff
@@ -23,17 +23,17 @@ def __init__(self):
         super(gfycat, self).__init__()
 
     def __fetch(self, url, param):
-        import urllib2
+        import urllib.request, urllib.error, urllib.parse
```

Comment:
I ran into at least one bug with this guy's class. I'll submit issue / pull-request on his repository sometime.